packages/core/src/input/Input.tsx

```diff
@@ -107,15 +107,15 @@ export const Input = forwardRef<HTMLDivElement, InputProps>(function Input(
     readOnly: formFieldReadOnly,
     validationStatus: formFieldValidationStatus,
   } = useFormFieldProps();
-  const { variant: controlWrapperVariant } = useControlWrapper();
+  const controlWrapper = useControlWrapper();
 
   const restA11yProps = {
     "aria-activedescendant": ariaActiveDescendant,
     "aria-expanded": ariaExpanded,
     "aria-owns": ariaOwns,
   };
 
-  const variant = controlWrapperVariant ?? variantProp;
+  const variant = controlWrapper?.variant ?? variantProp;
```

Comment:
The wrapper takes precedence over the prop someone has added to the input?  That's not the case for `disabled` or `readOnly` props.

Reply:
the wrapper is just for visual purposes (layout, variant). potentially we could add disabled and readonly to it to disable sections at a time, but i feel like that would add more into the context than intended

Reply:
thats interesting, tbh I never considered it that way round but I can see why it might be the case. I assumed wrapper > individual because you wouldn't want a random primary within a secondary wrapper if you're trying to force them all to be a certain variant. I'll raise this in FED meet up tomorrow if there's time and see what people think

+ to be honest, control wrapper could be changed to a stack layout potentially, for the layout, and we have something like a variant context instead..